src/main/java/org/jvnet/hudson/test/PluginAutomaticTestBuilder.java

```diff
@@ -86,8 +87,13 @@ public void testCliSanity() {
         public void testPluginActive() {
             String plugin = (String) params.get("artifactId");
             if (plugin != null) {
+                // did any plugin fail to start?
+                for (FailedPlugin fp : Jenkins.getInstance().getPluginManager().getFailedPlugins()) {
+                    throw new Error("Plugin "+fp.name+" failed to start", fp.cause);
+                }
+
                 PluginWrapper pw = Jenkins.getInstance().getPluginManager().getPlugin(plugin);
-                assertNotNull(pw);
+                assertNotNull(plugin+" failed to start", pw);
```

Comment:
`assertNotNull(plugin + " failed to start", pw);` in order to follow the same convention below